10_construct-database/63_construct_place_race-share-geography.qmd

```diff
@@ -89,17 +89,17 @@ read_csv(here("01_financial-well-being/city-debt-coll-shares-2021.csv")) |>
   )
 
 read_csv(here("06_neighborhoods/environment/data/final/environment_place_race-ethnicity_longitudinal.csv")) |>
-  mutate(subgroup = case_when(subgroup == "No Majority Race/Ethnicity" ~ "Mixed Race and Ethnicity", 
-                              subgroup == "Majority Non-White" ~ "Majority Households of Color",
+  mutate(subgroup = case_when(subgroup == "No Majority Race/Ethnicity" ~ "Mixed communities", 
+                              subgroup == "Majority Non-White" ~ "Communities of color",
                               TRUE ~ subgroup)) |>
```

Comment:
Similar comment to above - I think here you will have to add a line of code for White communities - I did not have to include Majority White, Non-Hispanic in the case_when because it was already written that way.